### DISCUSSION TITLE
Cross build (linux to windows) with Freetype

### DISCUSSION THREAD

**Discussion by JetSetIlly** [inkyblackness/imgui-go#174] [Q&A] [answered]
Has anyone managed to cross-build (from Linux to Windows) a static binary with Freetype rendering?

**Comment by JetSetIlly** [comment]
More background on this and a renewed plea for help:

I'm trying to build a windows binary with Freetype enabled. I can produce a Linux binary and a Mac binary with Freetype font rendering no problem. But Freetype on Windows is escaping me.

Has anyone managed to do this? (I'm using a mingw environment on Linux for cross compilation but I can't even manage a native compilation).

@neclepsio I see from this issue that you have managed compiling on Windows natively https://github.com/inkyblackness/imgui-go/issues/91 Which mingw components did you install?

**Reply by neclepsio** [reply]
If I remember correctly, I achieved this with msys2 - mingw64 toolchain, mingw-w64-x86_64-freetype and mingw-w64-x86_64-pkg-config packages.

**Reply by neclepsio** [reply]
But this was not a cross compilation, it was native on Windows.

**Reply by JetSetIlly** [reply]
Thanks. I've got it compiling now but it relies on the DLL. I have SDL statically linked but I can't seem to get freetype statically linked. But still, I've made some progress.

**Comment by JetSetIlly** [comment, marked as answer]
Fixed. It simply needs the "-static" flag in the CGO_LDFLAGS environment variable. I already had "-static-libgcc" and "-static-libstdc++" and this was sufficient for a static SDL compilation. But for freetype the additional flag is required.
